cylc/flow/option_parsers.py

```diff
@@ -49,12 +49,16 @@
 ID_MULTI_ARG_DOC = ('ID ...', 'Workflow/Cycle/Family/Task ID(s)')
 FULL_ID_MULTI_ARG_DOC = ('ID ...', 'Cycle/Family/Task ID(s)')
 
+SHORTLINK_TO_ICP_DOCS = "https://bit.ly/3MYHqVh"
+
 icp_option = Option(
     "--initial-cycle-point", "--icp",
-    metavar="CYCLE_POINT",
+    metavar="CYCLE_POINT or OFFSET",
     help=(
-        "Set the initial cycle point. "
+        "Set the initial cycle point."
```

Comment:
```suggestion
        "Set the initial cycle point. "
```
This needs the space.